packages-exp/auth-exp/test/helpers/integration/helpers.ts

```diff
@@ -18,7 +18,7 @@
 import { deleteApp, initializeApp } from '@firebase/app-exp';
 import { Auth, User } from '@firebase/auth-types-exp';
 
-import { getAuth } from '../../../index';
+import { getAuth } from '../../../'; // Use browser OR node dist entrypoint depending on test env.
```

Comment:
nit: two spaces between code and comment

Reply:
```suggestion
import { getAuth } from '../../../';  // Use browser OR node dist entrypoint depending on test env.
```